packages/cursorless-org-docs/src/docs/contributing/adding-a-new-language.md

```diff
@@ -1,3 +1,25 @@
+See description of facets in here,
```

Comment:
This change needs to be worked on. It looks like a bunch of separate thoughts. Did you really mean to commit it like this?

Reply:
No I did not, sorry! That was just my todo list, I am not used to how VSCode auto stages all changes

Reply:
Vscode doesn't auto stage anything for me. Might be a setting. That sounds dangerous :D